packages/overlay/src/overlay-stack.ts

```diff
@@ -164,9 +183,9 @@ export class OverlayStack {
 
         this.handlingResize = true;
         requestAnimationFrame(() => {
-            this.overlays.forEach((overlay) => {
+            for (const overlay of this.overlays) {
```

Comment:
What doe we get out of the `for..of` that `forEach` wasn't giving us here? I've not used it much yet, and not sure the specific benefits that come with it.